.github/workflows/publish.yml

```diff
@@ -2,20 +2,29 @@ name: Maven Publish
 
 on:
   release:
-    types: [published]
+    types: [ published ]
 
 jobs:
-  publish-artifacts:
-    runs-on: ubuntu-latest
+  publish:
+    runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
+      - name: Checkout repository
+        uses: actions/checkout@v2
         with:
-          java-version: 1.8
-      - name: Publish Artifacts
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.GC_AWS_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.GC_AWS_KEY }}
-        run: |
-          chmod +x gradlew
-          ./gradlew publish
\ No newline at end of file
+          fetch-depth: 0
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Setup JDK 1.8
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
+          distribution: 'adopt'
+      - name: Make Gradle wrapper executable
+        if: ${{ runner.os != 'Windows' }}
```

Comment:
We don't run this on Windows (this action doesn't have a os matrix either)

Reply:
its more of a sanity check for if we ever do